src/app/modules/login-module/login/login.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
+import { LoginService } from "../login.service";
+import { User } from "../user";
+
+@Component({
+  selector: 'ls-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+  private userEmail: string;
+  private userPassword: string;
+
+  constructor(private loginService: LoginService, private formBuilder: FormBuilder) {}
+
+  ngOnInit() {
+
+  }
+
+  loginForm = this.formBuilder.group({
+    email: ['', [
+      Validators.required,
+      Validators.minLength(4)
+    ]],
+    password: ['', [
+      Validators.required,
+      Validators.minLength(6)
+    ]]
+  })
+
```

Comment:
Please move code for form creation to separate private method `createForm` and run his in `ngOninit` method

Reply:
done